CASC/PortfolioMode.cpp

```diff
@@ -163,46 +177,26 @@ bool PortfolioMode::performStrategy(Shell::Property* property)
   // However, in SMTCOMP mode the fallback is universal for all
   // logics e.g. it's not very strong. Therefore, in SMTCOMP
   // mode we do main main_extra fallback fallback_extra
- 
-  Stack<Schedule> schedules;
+
```

Comment:
I think we "really" want a single schedule made up of the constituent main/fallback parts. Then we can call `getExtraSchedules` once with the single mega-schedule when we need more.